### ISSUE TITLE
Enter Card Serial Number - Validation Message Styling / Formatting Issues

### ISSUE THREAD

**Issue by jeznorth** [bcgov/bc-wallet-mobile#3490]
The validation message is incorrectly spaced and does not meet accessibility standards.

## Expected Behaviour
- Spacing / colour should be consistent with design specification.
- Input label should appear as 'bold' (e.g. 'Enter serial number'). 
- Page title is incorrect size.

### Form Design Specifications
https://www.figma.com/design/LaNyNJGxNMsxlqUCNTWy1X/BCSC-V4---Phase-1?node-id=2911-210&t=QG4V5DCR1i4T2g0Q-1

### Page Formatting (Page title)
https://www.figma.com/design/LaNyNJGxNMsxlqUCNTWy1X/BCSC-V4---Phase-1?node-id=1638-10502&t=uosDz4ybVookzD4b-1

## Screenshot
<img width="140" height="400" alt="Image" src="https://github.com/user-attachments/assets/7a120cb1-7970-494a-a5fd-b53d48549aa8" />

## Additional Notes
It might be worth looking at the 'Account Nickname' screen to ensure it's using a standardized pattern.
